scrapers/AdultFilmIndex.yml

```
@@ -0,0 +1,75 @@
+name: AdultFilmIndex
+sceneByURL:
+  - action: scrapeJson
+    url:
+      - https://adultfilmindex.com/api/v1/stash/scene/
+    scraper: sceneScraper
+sceneByName:
+  action: scrapeJson
+  queryURL: https://adultfilmindex.com/api/v1/stash/scene_search/{}
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeJson
+  queryURL: "{url}"
+  scraper: sceneScraper
+sceneByFragment:
+  action: scrapeJson
+  queryURL: https://adultfilmindex.com/api/v1/stash/scene_fragment_search/{filename}/{oshash}
+  scraper: sceneScraper
+movieByURL:
+  - action: scrapeJson
+    url:
+      - https://adultfilmindex.com/movie/
+    scraper: movieScraper
+    queryURL: https://adultfilmindex.com/api/v1/stash/movie/{url}
+    queryURLReplace:
+      url:
+        - regex: '.+/movie/(.*)/(.*).+'
+          with: "${1}"
+jsonScrapers:
+  sceneSearch:
+    scene:
+      Title: data.#.title
+      Date: data.#.release_date
+      Image: data.#.thumbnail
+      URL: data.#.url
+      Details: data.#.description
+  sceneScraper:
+    scene:
+      Title: data.title
+      Details: data.description
+      URL: data.url
+      Image: data.thumbnail
+      Date:
+        selector: data.movie.release_date
+        postProcess:
+          - parseDate: 2006-01-02
```

it seems that we need some pp to remove the `T.*` part of the date
```suggestion
          - replace:
              - regex: T.*$
                with: ""
          - parseDate: 2006-01-02
```